.github/workflows/dev_builds.yml

```diff
@@ -5,10 +5,14 @@ on:
     branches:
       - main
       - 'release-v[0-9]+.[0-9]+'
+  pull_request:
+    branches:
+      - main
+      - 'release-v[0-9]+.[0-9]+'
```

Comment:
I think this was just for testing, and should be removed?

Reply:
If you want to build multiplatform images for each PR, you need to add it here: https://github.com/SumoLogic/tailing-sidecar/blob/main/.github/workflows/pull_requests.yml. What you've done here is make us build *and push* the image for each PR, which we definitely don't want.

Reply:
Ok, I will remove it.